browser/js/background.js

```diff
@@ -2,10 +2,37 @@
 
 const KBNM_HOST = "io.keybase.kbnm";
 
+// Set the default badge color
+chrome.browserAction.setBadgeBackgroundColor({
+  color: "#3dcc8e"
+});
+
 // Relay extension messages to native messages.
 chrome.runtime.onMessage.addListener(function(msg, sender, sendResponse) {
+  if (sender.tab) {
+    // Reset the tab state with each query
+    chrome.browserAction.setBadgeText({
+      text: "",
+      tabId: sender.tab.id,
+    });
+  }
+
+  const isPassive = msg["method"] === "passivequery";
+  if (isPassive) {
+    // TODO: This will be a special method at some point, but for now we're
+    // prototyping this feature with the normal query method.
+    msg["method"] = "query";
+  }
+
   chrome.runtime.sendNativeMessage(KBNM_HOST, msg, function(r) {
     if (r) {
+      if (isPassive && r.status === "ok") {
+        // Set badge
+        chrome.browserAction.setBadgeText({
+          text: "✓",
```

Comment:
This is a temporary prototype using badge text until we get a proper icon to swap out with.